models/accountSchema.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+const accountSchema = new mongoose.Schema(
+  {
+    title: {
+      type: String,
+      required: [true, "Please enter the title"],
+    },
+    description: {
+      type: String,
+    },
+    currency: {
+      type: String,
+    },
+    createdBy: {
+      type: mongoose.Types.ObjectId,
+      ref: "User",
+    },
+  },
+  {
+    timestamps: true,
```

Comment:
Dont you need userId to link account with user?

Reply:
Oh... you use createdBy field? It's better to give it name like 'userId'

Reply:
yes i'll rename it.